effectiveOfficeBackend/src/main/kotlin/office/effective/model/IntegrationModel.kt

```diff
@@ -0,0 +1,18 @@
+package office.effective.model
+
+import java.util.*
+
+data class IntegrationModel(var name: String, var valueStr: String) {
+    var id: UUID? = null
+    var iconUrl: String = ""
+
+    constructor(
+        _id: UUID,
+        _name: String,
+        _valueStr: String,
+        _iconUrl: String
```

Comment:
You don`t need to use underlines here, you use "this" anyway.

Reply:
Read about constructors in kotlin, you can write this in one line, just using primary constructor, not secondary